packages/adapter-neon/package.json

```
@@ -39,9 +39,9 @@
     "postgres-array": "3.0.2"
   },
   "devDependencies": {
-    "@neondatabase/serverless": "0.6.0"
+    "@neondatabase/serverless": "0.7.1"
   },
   "peerDependencies": {
-    "@neondatabase/serverless": "^0.6.0"
+    "@neondatabase/serverless": "^0.6.0 || ^0.7.0"
```

```suggestion
    "@neondatabase/serverless": ">= 0.6.0 < 1"
```
I suggest we switch to this or `>= 0.6.0`

---

[Internal Slack discussion](https://prisma-company.slack.com/archives/C04TW4A2H8C/p1704875058247519)

I found out that it’s very restrictive (see https://semver.npmjs.com/ for trying it out) because the caret does not behave as one could expect when the major is 0 :sweat_smile:
This is expected when using semver with npm but not well known I think:

> Allows changes that do not modify the left-most non-zero element in the [major, minor, patch] tuple. In other words, this allows patch and minor updates for versions 1.0.0 and above, patch updates for versions 0.X >=0.1.0, and no updates for versions 0.0.X.

https://github.com/npm/node-semver#caret-ranges-123-025-004

---

Ignoring my suggestion for now and going for Renovate's